<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageBms</class>
 <widget class="QWidget" name="PageBms">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>8</number>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Voltages</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCustomPlot" name="plotCells" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Temperatures</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCustomPlot" name="plotTemp" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Temp Plot</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="BattTempPlot" name="tempPlot" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTableWidget" name="valTable">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>25</number>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>25</number>
         </attribute>
         <row>
          <property name="text">
           <string>Total Voltage</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Charge Input Voltage</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Input Current</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Input Current BMS IC</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Ah Counter</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Wh Counter</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Power</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>State of Charge (SoC)</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>State of Health (SoH)</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Temp Cell Max</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Humidity</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Pressure</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Temp Hum Sensor</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Ah Charge Total</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Wh Charge Total</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Ah Discharge Total</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Wh Discharge Total</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Value</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="4" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="5" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="6" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="7" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="8" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="9" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="10" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="11" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="12" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="13" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="14" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="15" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="16" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QPushButton" name="resetAhButton">
           <property name="text">
            <string>Reset Ah</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/res/icons/Restart-96.png</normaloff>:/res/icons/Restart-96.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="resetWhButton">
           <property name="text">
            <string>Reset Wh</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/res/icons/Restart-96.png</normaloff>:/res/icons/Restart-96.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QPushButton" name="balOnButton">
             <property name="text">
              <string>Bal On</string>
             </property>
             <property name="icon">
              <iconset resource="../res.qrc">
               <normaloff>:/res/icons/Circled Play-96.png</normaloff>:/res/icons/Circled Play-96.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="balOffButton">
             <property name="text">
              <string>Bal Off</string>
             </property>
             <property name="icon">
              <iconset resource="../res.qrc">
               <normaloff>:/res/icons/Stop-96.png</normaloff>:/res/icons/Stop-96.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="zeroCurrentButton">
           <property name="text">
            <string>Cal Zero Current</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/res/icons/Refresh-96.png</normaloff>:/res/icons/Refresh-96.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QPushButton" name="chgEnButton">
             <property name="text">
              <string>Chg En</string>
             </property>
             <property name="icon">
              <iconset resource="../res.qrc">
               <normaloff>:/res/icons/Circled Play-96.png</normaloff>:/res/icons/Circled Play-96.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="chgDisButton">
             <property name="text">
              <string>Chg Dis</string>
             </property>
             <property name="icon">
              <iconset resource="../res.qrc">
               <normaloff>:/res/icons/Stop-96.png</normaloff>:/res/icons/Stop-96.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>widgets/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BattTempPlot</class>
   <extends>QWidget</extends>
   <header>widgets/batttempplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
